web/src/compose_tooltips.js

```diff
@@ -104,11 +105,22 @@ export function initialize() {
         target: ".send-control-button",
         delay: LONG_HOVER_DELAY,
         placement: "top",
-        onShow() {
+        onShow(instance) {
             // Don't show send-area tooltips if the popover is displayed.
             if (popover_menus.is_scheduled_messages_popover_displayed()) {
                 return false;
             }
+            if (instance.reference.id === "compose-drafts-button") {
+                const count =
+                    instance.reference.querySelector(".compose-drafts-count > .count")
+                        .textContent || 0;
+                // Explain that the number in brackets is the number of drafts for this conversation.
+                const draft_count_msg = $t(
+                    {defaultMessage: "{count} draft{plural} for this conversation"},
+                    {count, plural: count === "1" ? "" : "s"},
```

Comment:
You can't internationalize strings like this -- appending `s` suffixes only works in English.

https://zulip.readthedocs.io/en/latest/translating/internationalization.html is missing documentation on this, but you need to use the ICU message format: https://unicode-org.github.io/icu/userguide/format_parse/messages/

Reply:
Here's a link to the [CZO discussion](https://chat.zulip.org/#narrow/stream/6-frontend/topic/handling.20possibly.20plural.20nouns/near/1683633) on the same, that I think might help. :)

Reply:
All those links were helpful, thank you so much!

I've updated this accordingly, it should work now?